seqr/views/utils/variant_utils.py

```diff
@@ -131,6 +132,16 @@ def _saved_variant_genes_transcripts(variants):
     return genes, transcripts, family_genes
 
 
+def get_omim_intervals_query(variants):
+    chroms = {v['chrom'] for v in variants if v.get('svType')}
+    return Q(phenotype_mim_number__isnull=False, gene__isnull=True, chrom__in=chroms)
+
+
+def _get_omim_intervals(variants):
+    omims = Omim.objects.filter(get_omim_intervals_query(variants))
+    return {o['phenotypeMimNumber']: o for o in get_json_for_queryset(omims)}
```

Comment:
I just went to the `Omim` model definition and found
```
    class Meta:
        # ('mim_number', 'phenotype_mim_number') is not unique - for example ('124020', '609535')
        unique_together = ('mim_number', 'phenotype_mim_number', 'phenotype_description')
``` 
Do you think we'll get multiple models with the same `phenotypeMimNumber`, and would that mean not returning all possible omims?

Reply:
oh yeah, good catch. I will look into this further.

Reply:
Okay so this is an issue, but its an issue already in how we handle existing omim conditions associated with genes for reporting and discovery. I would like to fix it but I think its out of scope for this PR, so my proposed solution is that this PR goes forward with the current logic here for collapsing multiple phenotype_mim_number into one for conditions with and without genes, and then I will put up a follow up PR later today that handles this correctly everywhere in seqr

Reply:
sounds great!